test/steps/crypto/1_given.js

```diff
@@ -130,6 +136,48 @@ export function aPassphraseWithPrivateKeyAndPublicKeyAndAddress() {
 	this.test.ctx.address = address;
 }
 
+export function numberOfAccountsAsOneAndPassphraseWithPrivateKeyAndPublicKeyAndAddress() {
```

Comment:
numberOfAccounts seems to be taken from the title, and non-one

Reply:
this function's name has been changed to numberOfAccountsAndPassphraseWithPrivateKeyAndPublicKeyAndAddress.